dotnet/src/Microsoft.AutoGen/RuntimeGateway.Grpc/Abstractions/AgentsRegistryState.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// AgentsRegistryState.cs
+using System.Collections.Concurrent;
+using Microsoft.AutoGen.Contracts;
+using Microsoft.AutoGen.Protobuf;
+
+namespace Microsoft.AutoGen.RuntimeGateway.Grpc.Abstractions;
+public class AgentsRegistryState
+{
+    public ConcurrentDictionary<string, HashSet<string>> AgentsToEventsMap { get; set; } = new ConcurrentDictionary<string, HashSet<string>>();
```

Comment:
No longer relevant, remove?

Reply:
no this is relevant this is the object for the registry that gets written to Orleans.

Reply:
or do you mean just the events part?

Reply:
just the events

Reply:
removed